### PR TITLE
BZ2003722: Add bare metal as platform for SR-IOV Network Operator

### PR DESCRIPTION
**Pull request by jboxman-rh**
- https://bugzilla.redhat.com/show_bug.cgi?id=2003722

@zshi-redhat, is this limited to installer-provisioned or user-provisioned bare metal or does it work with both?

Thanks!

### CODE REVIEW

**Comment by jboxman-rh** [comment]
@zshi-redhat, ping

File: modules/nw-sriov-supported-platforms.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+//
+// * networking/hardware_networks/about-sriov.adoc
+
+[id="nw-sriov-supported-platforms_{context}"]
+= Supported platforms
+
+The SR-IOV Network Operator is supported on the following platforms:
+
+- Bare metal
+
```
Comment (zshi-redhat) [inline modules/nw-sriov-supported-platforms.adoc:11]: Please also add `OpenStack` platform.

Comment (jboxman-rh) [reply modules/nw-sriov-supported-platforms.adoc]: @zshi-redhat done!

File: networking/hardware_networks/about-sriov.adoc [hunk 2]
```diff
@@ -15,7 +15,7 @@ The SR-IOV network device driver for the device determines how the VF is exposed
 * `netdevice` driver: A regular kernel network device in the `netns` of the container
 * `vfio-pci` driver: A character device mounted in the container
 
-You can use SR-IOV network devices with additional networks on your {product-title} cluster for application that require high bandwidth or low latency.
+You can use SR-IOV network devices with additional networks on your {product-title} cluster installed on bare metal infrastructure for application that require high bandwidth or low latency.
```
Comment (jeana-redhat) [inline networking/hardware_networks/about-sriov.adoc:18]:
```suggestion
You can use SR-IOV network devices with additional networks on your {product-title} cluster installed on bare metal infrastructure for applications that require high bandwidth or low latency.
```

Comment (jeana-redhat) [reply networking/hardware_networks/about-sriov.adoc:18]: Also, should this sentence mention RHOSP as well as bare metal like in the module?

**Review by jeana-redhat — COMMENTED** [review]
Just saw an older typo, and have a question. Looks good overall!

**Pull request merged**